newui/docs/_config.yml

```diff
@@ -0,0 +1,218 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your entire site, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# `jekyll serve`. If you change this file, please restart the server process.
+
+#
+hudi_style_skin          : "hudi"
+
+version : &version "0.5.1-SNAPSHOT"
+
+previous_docs:
+  latest: /docs/quick-start-guide.html
+  0.5.0-incubating: /versions/0.5.0-incubating/docs/quick-start-guide.html
+
+
+# Site Settings
+locale                   : "en-US"
+title                    : "Apache Hudi"
+title_separator          : "-"
+subtitle                 : *version
+description              : "Apache Hudi Stands for Hadoop Upserts and Incrementals to manage the storage of large analytical datasets on HDFS."
```

Comment:
Reword : Apache Hudi ingests & manages storage of large analytical datasets over DFS (HDFS or cloud stores).

Please keep the landing page consistent with what we have. We can deal with content changes separately

Reply:
Done.